tools/flecstan/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# -*- CMake -*-
+
+project(flecstan LANGUAGES CXX)
+cmake_minimum_required(VERSION 3.8)
+
+# Use cmake modules from cinch
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../cinch/cmake)
+
+# C++
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pedantic -Wall")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti -O2 -s")
+
+# Packages
+find_package(Clang REQUIRED COMPONENTS Tooling)
+
+# Include directories
+include_directories(src ${CLANG_INCLUDE_DIRS} ${LLVM_INCLUDE_DIRS})
```

Comment:
There is no `find_package(LLVM)` anyhow, hence `${LLVM_INCLUDE_DIRS}` is empty and should be dropped.

Reply:
Maybe not. Consider the following (recommended for everyone; this has proven to be highly useful for me):

     # Print all variable names
     # From https://stackoverflow.com/questions/9298278/
     get_cmake_property(_variableNames VARIABLES)
     list (SORT _variableNames)
     foreach (_variableName ${_variableNames})
         message(STATUS "${_variableName}=${${_variableName}}")
     endforeach()

Place at the bottom of your CmakeLists.txt. Then I run:

     cmake . | grep -i llvm_include_dir

And see this output:

     -- LLVM_INCLUDE_DIR=/usr/local/include
     -- LLVM_INCLUDE_DIRS=/usr/local/include

in spite of not having a find_package(LLVM).

I believe that the inclusion of Clang automatically triggers CMake to include LLVM as well. In fact, at one point I had a find_package(LLVM), and removed it after discovering that it apparently wasn't needed (and I thought there was a fighting change that changing *something* might solve this mysterious problem of cmake not finding some Clang libraries during one of the CI builds.)

Reply:
I guess, that got inherited by flecsi project, so ok.

If you want to allow a stand-alone build, I would add `find_packge(LLVM ..)` back.

Reply:
What I described above works here even if I do a standalone build, entirely outside of the flecsi project. Perhaps I'll put the find_package back in for safety's sake.